examples/optimizer_wrappers.py

```diff
@@ -0,0 +1,60 @@
+"""
+PyTorch version: https://github.com/pytorch/examples/blob/master/mnist/main.py
+TensorFlow version: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist.py
```

Comment:
This is copy/pasted from `mnist.py`, but I'm not sure we'd want to repeat it here? Or do you want to replace the `mnist.py` example with this one?

Reply:
Not exactly sure what to do with this example just yet, for now I just kinda meant it as a demonstration for the reviewers of the PR :-).